app/views/providers/online_bankings/show.html.erb

```diff
@@ -1,17 +1,23 @@
+<% content_for(:page_title) { t('.h1-heading') } %>
 <% content_for(:navigation) { provider_back_link } %>
 
 <%= render partial: 'shared/forms/error_summary', locals: { model: @form } %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+
```

Comment:
I don't think this fieldset is needed - the current affect is for it to duplicate the heading on the page: 

<img width="658" alt="screen shot 2018-12-19 at 14 06 05" src="https://user-images.githubusercontent.com/28729201/50225025-48edcd80-0397-11e9-8cd1-860c1e601a2b.png">

Reply:
Indeed. The `govuk_collection_radio_buttons` component generates the`<legend>` via the `title` option